src/app/(app)/funds/components/FundFormBuilder.tsx

```diff
@@ -47,7 +52,21 @@ export const FundFormBuilder = () => {
   const { isValid } = form.formState;
 
   function onSubmit(values: z.infer<typeof FundSchema>) {
-    return values;
+    try {
+      addFunds(values);
+      toast({
+        title: "Success",
+        description: "Fund added successfully",
+      });
+      router.push(Routes.Fund);
```

Comment:
can we remove this because we are using redirect in server action ?

Reply:
@Mubashar1009 My mistake; I have also implemented a redirect in the server action, so there's no need for this.